### PR TITLE
final commit

### PR DESCRIPTION
**Pull request by nehalskdev**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  min-width: 400px;
+  min-height: 100vh;
+
+  display: grid;
```
Comment (ManSOorcode) [inline style.css]: remove grid from here because you override it by display flex

**Review by ManSOorcode — CHANGES_REQUESTED** [review]

**Review by murtazshaikh — APPROVED** [review]
Approved

File: style.css (hunk 1 above)
Comment (nehalskdev) [reply style.css]: changes made please check @ManSOorcode

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <img src="image.png" alt="image logo" />
+      </div>
+      <p class="title-p">Images</p>
+      <div class="size-file">
+        <p>257.4 GB</p>
+        <p>12,847 files</p>
+      </div>
+      <div class="loading"></div>
+    </div>
+    <div class="frem">
```
Comment (Abbas4tech) [inline index.html:22]: Add some meaningfull names for class

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      <p class="title-p">Videos</p>
+      <div class="size-file">
+        <p>4 GB</p>
+        <p>125 files</p>
+      </div>
+      <div class="loading-1"></div>
+    </div>
+    <div class="frem">
+      <div class="logo">
+        <img src="image.png" alt="image logo" />
```
Comment (Abbas4tech) [inline index.html:35]: What does `alt` explain about the image??? i.e `image logo` ??

File: index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      <p class="title-p">Images</p>
+      <div class="size-file">
+        <p>257.4 GB</p>
+        <p>12,847 files</p>
+      </div>
+      <div class="loading"></div>
+    </div>
+    <div class="frem">
+      <div class="logo">
+        <img src="image.png" alt="image logo" />
```
Comment (Abbas4tech) [inline index.html:24]: Same here

File: style.css [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+  width: 22rem;
+  height: 91px;
+
+  background: white;
+  box-shadow: 12px 11px 20px #00000f;
+  border-radius: 21px;
+
+  display: flex;
+  justify-content: flex-start;
```
Comment (Abbas4tech) [inline style.css]: Do we need this ?m

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
Need some refactor

File: index.html (hunk 3 above)
Comment (nehalskdev) [reply index.html:35]: the image added here is the symbol of image in any digital gallery hence i have kept it as image logo.

File: index.html (hunk 4 above)
Comment (nehalskdev) [reply index.html:24]: as mentioned in my above comment. i would appreciate if you can suggest me some insightful ```alt``` for the image used.

File: style.css (hunk 5 above)
Comment (nehalskdev) [reply style.css]: thanks I am taking it down.

**Pull request merged**